app/controllers/application_controller.rb

```diff
@@ -1,3 +1,15 @@
 class ApplicationController < ActionController::Base
+  
   protect_from_forgery with: :exception
+  before_action :authenticate_volunteer!
+  before_action :configure_permitted_parameters, if: :devise_controller?
+protected
```

Comment:
indent this please. And put a space on top and bottom to separate it from the other codes